.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
Just wandering if this should not already be covered by git

Reply:
Still could be used as a guide by many editors/IDEs without additional setup required.